app/presenters/filtered_editions_presenter.rb

```diff
@@ -1,7 +1,59 @@
 # frozen_string_literal: true
 
 class FilteredEditionsPresenter
+  def initialize(states_filter: [], assigned_to_filter: nil, format_filter: nil, title_filter: nil)
+    @states_filter = states_filter || []
+    @assigned_to_filter = assigned_to_filter
+    @format_filter = format_filter
+    @title_filter = title_filter
+  end
+
+  def available_users
+    User.enabled.alphabetized
+  end
+
   def editions
-    Edition.all
+    result = editions_by_format
+    result = apply_states_filter(result)
```

Comment:
might be difficult to do anything about it, not sure if I like passing result and reading result.

Reply:
Yeah, I get what you mean, but I'm not sure how else to achieve this really.